### PR TITLE
Pootle formats pootle_fs hookup

### PR DESCRIPTION
**Pull request by phlax**
add `filetypes` support to pootle_fs

### CODE REVIEW

File: pootle/apps/pootle_fs/management/commands/init_fs_project.py [hunk 1]
```diff
@@ -94,11 +93,15 @@ def handle(self, **options):
             project = Project.objects.create(
                 code=code,
                 fullname=name,
-                localfiletype=options['file_type'],
                 treestyle='none',
                 checkstyle=options['checkstyle'],
-                source_language=source_language
-            )
+                source_language=source_language)
+            for filetype in options["filetypes"] or ["po"]:
```
Comment (dwaynebailey) [inline pootle/apps/pootle_fs/management/commands/init_fs_project.py]: Since you already set 'po' as a default, you shoudn't need to `or` here at all.

Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/init_fs_project.py]: it was my lack of argparse-fu - it was failing every way i tried it - apart from this

Comment (unho) [inline pootle/apps/pootle_fs/management/commands/init_fs_project.py]: Didn't `default=["po"]` in `parser.add_argument()` above work?

Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/init_fs_project.py]: no, with append it was making [["po"]]

im sure there is a better way tho

**Comment by unho** [comment]
lgtm since argparse seems not to be able to handle specifying a default when using `append`.

File: pootle/apps/pootle_fs/management/commands/init_fs_project.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_fs/management/commands/init_fs_project.py]: Just for the record this is not what I got, but I hit another different issue (https://bugs.python.org/issue16399). My stance is to land the PR as it is.

**Pull request merged**